src/System.Management.Automation/engine/runtime/Operations/ArrayOps.cs

```diff
@@ -52,7 +52,7 @@ internal static T[] Multiply<T>(T[] array, uint times)
 
             if (times == 0 || array.Length == 0)
             {
-                return new T[0]; // don't use Utils.EmptyArray, always return a new array
+                return Array.Empty<T>();
```

Comment:
@iSazonov Perhaps we should suppress `CA1825` here, if there is still a rationale for always returning a new array instance. `Utils.EmptyArray` was removed in your PR #9042.

Reply:
Yes, please revert and suppress.

Reply:
I will revert and suppress, but I was wondering what reason we do not to use the static empty array?

Reply:
I don't remember. Perhaps it is used in comparisons.

Reply:
@PaulHigin Must we always return a new instance, rather then using the static empty array?